### ISSUE TITLE
Incorporate "proposed" units from EIA into matches when they show up in CAMD before EIA?

### ISSUE THREAD

**Issue by j-tafoya** [USEPA/camd-eia-crosswalk#22]

**Comment by grgmiller** [comment]
Yes, please. This would actually be helpful for our hourly emissions project, which uses pudl's `epa_crosswalk` module (https://github.com/catalyst-cooperative/pudl/blob/main/src/pudl/analysis/epa_crosswalk.py) to assign a "subplant ID" to each unit in CEMS.
